examples/blocks/src/custom/index.html

```diff
@@ -77,14 +80,9 @@
                 const name = metadata.value;
                 const span = document.createElement("span")
                 const img = document.createElement("img");
-                img.onload = () => {
-                    img.onload = undefined;
-                    const data = getBase64Image(img);
-                    img.src = data;
-                }
                 img.setAttribute("crossorigin", "anonymous")
-                img.setAttribute("src", `http://perspective.finos.org/img/flags/${states[clean_name].toLowerCase()}.png`);
-   
+                img.setAttribute("src", `http://flags.ox3.in/svg/us/${states[clean_name].toLowerCase()}.svg`);
```

Comment:
I think you left some debugging in

Reply:
To be clear source was not mine, and changed it because hotlinking from finos.org was not allowed so flags were not rendered. Will include the flags as a local resource; the SVG files are in the public domain anyway